scripts/cat/cats.py

```diff
@@ -279,66 +291,98 @@ def __init__(self,
         if self.gender is None:
             self.gender = choice(["female", "male"])
         self.g_tag = self.gender_tags[self.gender]
+        
+        '''if self.genderalign == "":
+            self.genderalign = self.gender'''
 
-        # These things should only run when generating a new cat, rather than loading one in.
+         # These things should only run when generating a new cat, rather than loading one in.
         if not loading_cat:
             # trans cat chances
+            self.genderalign = self.gender
             trans_chance = randint(0, 50)
             nb_chance = randint(0, 75)
-            if self.gender == "female" and not self.status in ['newborn', 'kitten']:
-                if trans_chance == 1:
-                    self.genderalign = "trans male"
-                elif nb_chance == 1:
-                    self.genderalign = "nonbinary"
+            if game:
+                if game.settings["they them default"] is True:
+                    they_them_default_setting = True
+                    print("Game is using they/them pronouns")
                 else:
-                    self.genderalign = self.gender
-            elif self.gender == "male" and not self.status in ['newborn', 'kitten']:
-                if trans_chance == 1:
-                    self.genderalign = "trans female"
-                elif nb_chance == 1:
-                    self.genderalign = "nonbinary"
+                    they_them_default_setting = False
+                    print("Game is not using they/them pronouns")
+
+                if they_them_default_setting is True:
```

Comment:
This works well, but I would like to clean it up and make it a bit more readable, and less of a maze of if-statements. Not high priority.